apps/src/sites/studio/pages/scripts/edit.js

```diff
@@ -78,6 +78,9 @@ export default function initPage(scriptEditorData) {
           scriptData.weeklyInstructionalMinutes
         }
         isMigrated={scriptData.is_migrated}
+        includeStudentFacingLessonPlans={
+          scriptData.includeStudentFacingLessonPlans
```

Comment:
Can you move the default value setting out here instead of in the state for consistency?